pkg/config/manager.go

```diff
@@ -60,6 +60,7 @@ func (m *Manager) LoadConfig() (*Config, error) {
 	if err = yaml.Unmarshal(payload, &cfg); err != nil {
 		return nil, err
 	}
+	LoadedConfig = cfg
```

Comment:
I don't see `LoadedConfig` being used anywhere else besides here and being declared in `pkg/config/config.go`. Was this leftover from something else?

Reply:
It is used in `NewHostPathDriver` and `Run` in hostpath.go

Reply:
Perhaps the use was in a separate commit from the one that has this reference (a guess, did not investigate to confirm)

Reply:
I loaded all commits and used Ctrl-F in the browser. Not a big deal.